### DISCUSSION TITLE
BigQuery Emulator using goccy / bigquery-emulator

### DISCUSSION THREAD

**Discussion by addijhaq** [testcontainers/testcontainers-java#6799] [Ideas]
Interested in what it would take to create a new testcontainer module for a prexisting repo.

[goccy](https://github.com/goccy)/[bigquery-emulator](https://github.com/goccy/bigquery-emulator) 
is regularly maintained and used to mock out bigquery. Since there seems to be very little progress being made with GCloud Module BigQuery emulation, it would be beneficial if we could utilize this pre-existing emulator in the meantime. 

Instead of pushing this off on the testcontainers team, I'd be happy to help spearhead the initiative to help implement a testcontainers module.  (Docker images already exist for this repo)

I have incentive to do so because my current paid work requires the use of bigquery and its cost is non trivial to utilize a real, working version of bigquery just for testing purposes.

**Comment by eddumelendez** [comment]
Hi @addijhaq, sorry for the delay. We have decided to add it. See https://github.com/testcontainers/testcontainers-java/pull/7324

Your feedback would be very welcome.
